web/styles/zulip.css

```diff
@@ -1240,10 +1246,9 @@ td.pointer {
 
     .star_container {
         &:not(.empty-star) {
-            color: hsl(106deg 77% 29%);
-
-            /* Opacity/visibility as though the message is hovered. */
-            opacity: 1 !important;
+            /* Color, opacity, and visibility, as though the message is hovered. */
+            color: var(--color-message-action-interactive);
+            opacity: 1;
```

Comment:
Do we want to change the hover color for the message timestamp to be more consistent with these?

Reply:
That makes very good sense to me.

Reply:
One lingering point is that the hover styles for the EDITED messages are set with opacity. They don't stick out like the dark-mode timestamp hover does, but I'd prefer to get away from that and set actual colors on things.